soc/riscv/riscv-privilege/telink_b9x/Kconfig.soc

```diff
@@ -52,6 +56,10 @@ config SOC_RISCV_TELINK_B92
 
 endchoice
 
+config PMP_SLOTS
+	depends on RISCV_PMP
```

Comment:
```
if RISCV_PMP

config PMP_SLOTS
	int "Number of PMP slots"
	default 8
	help
	  This is the number of PMP entries implemented by the hardware.
	  Typical values are 8 or 16.
```
The dependency is already introduced. We just changing default
See soc/riscv/riscv-privilege/virt/Kconfig.defconfig.series